cpu/nrf52/include/cpu_conf.h

```diff
@@ -57,12 +62,15 @@ extern "C" {
  */
 #define FLASHPAGE_SIZE                  (4096U)
 
-#if defined(CPU_MODEL_NRF52832XXAA)
+#if defined(CPU_MODEL_NRF52811XXAA)
+#define FLASHPAGE_NUMOF                 (48U)
+#elif defined(CPU_MODEL_NRF52832XXAA)
 #define FLASHPAGE_NUMOF                 (128U)
 #elif defined(CPU_MODEL_NRF52840XXAA)
 #define FLASHPAGE_NUMOF                 (256U)
 #endif
 
+
```

Comment:
That extra new line isn't needed.

Reply:
There are also other formatting issues. Just seen it on travis. Will change it.